include/onnxruntime/core/providers/cuda/cuda_context.h

```diff
@@ -28,38 +28,70 @@
   cudnnHandle_t cudnn_handle = {};
   cublasHandle_t cublas_handle = {};
   OrtAllocator* deferred_cpu_allocator = {};
+  // below are cuda ep options
+  int16_t device_id = 0;
+  bool has_user_compute_stream = false;
+  size_t gpu_mem_limit = 0;
+  int32_t arena_extend_strategy = 0;
+  int32_t cudnn_conv_algo_search = 0;
+  bool do_copy_in_default_stream = true;
+  void* gpu_external_alloc = {};
+  void* gpu_external_free = {};
+  void* gpu_external_empty_cache = {};
+  bool cudnn_conv_use_max_workspace = true;
+  bool enable_cuda_graph = false;
+  bool cudnn_conv1d_pad_to_nc1d = false;
+  bool tunable_op_enable = false;
+  bool tunable_op_tuning_enable = false;
+  int32_t tunable_op_max_tuning_duration_ms = 0;
+  bool enable_skip_layer_norm_strict_mode = false;
+  bool prefer_nhwc = false;
+  bool use_ep_level_unified_stream = false;
```

Comment:
>bool use_ep_level_unified_stream = false; [](http://example.com/codeflow?start=2&length=41)

same as above, we don't need to expose it, as its usage already covered in cuda stream